examples/cat.cpp

```diff
@@ -1,6 +1,6 @@
-#include "eff.hpp"
-#include "stream.hpp"
-#include "task.hpp"
+// #include <cxxeffect/eff.hpp>
```

Comment:
If it's not needed, it should just be removed. Alternatively, perhaps `eff.hpp` should import both `stream` and `task` for you?

Reply:
This was an oversight; I'll remove it

Reply:
What do you think about `eff.hpp` including all of the library headers by default? That would make SFH generation easier too, wouldn't it?

Reply:
> If it's not needed, it should just be removed. Alternatively, perhaps `eff.hpp` should import both `stream` and `task` for you?

I removed this include!